Pipfile

```diff
@@ -17,9 +17,11 @@ google-cloud-storage = "*"
 pandas = "*"
 matplotlib = "*"
 slackclient = "*"
+pillow = "*"
+pyyaml = "*"
```

Comment:
Were these explicitly installed due to dependency issues but since they are also using the latest version I don't get why these were added?